app/models/company.rb

```diff
@@ -1,3 +1,7 @@
 class Company < ApplicationRecord
   validates :name, presence: :true
+
+  def generate_access_key
+    SecureRandom.uuid
+  end
```

Comment:
This is consistent with the way we generate access keys for [users](https://github.com/kevin-j-m/ivory-tower/blob/d678c30279162063729af6b9cb6ef0a330c52616/app/models/user.rb#L4). How do you feel about DRYing these concerns up to a single place?